lang/src/org/partiql/lang/compiler/PartiQLCompilerPipeline.kt

```diff
@@ -0,0 +1,59 @@
+package org.partiql.lang.compiler
+
+import com.amazon.ion.IonSystem
+import com.amazon.ion.system.IonSystemBuilder
+import org.partiql.lang.CompilerPipeline
+import org.partiql.lang.domains.PartiqlAst
+import org.partiql.lang.domains.PartiqlPhysical
+import org.partiql.lang.errors.PartiQLException
+import org.partiql.lang.eval.PartiQLStatement
+import org.partiql.lang.planner.PartiQLPlanner
+import org.partiql.lang.planner.PartiQLPlannerBuilder
+import org.partiql.lang.syntax.Parser
+import org.partiql.lang.syntax.SqlParser
+
+class PartiQLCompilerPipeline(
+    private val parser: Parser,
+    private val planner: PartiQLPlanner,
+    private val compiler: PartiQLCompiler
+) {
+
+    companion object {
+
+        private val ION_STANDARD = IonSystemBuilder.standard().build()
+
+        @Deprecated(
+            message = "To be removed in 1.0 release",
+            replaceWith = ReplaceWith("standard"),
+            level = DeprecationLevel.WARNING
+        )
+        @JvmStatic
+        fun legacy(ion: IonSystem = ION_STANDARD): CompilerPipeline.Builder = CompilerPipeline.builder(ion)
```

Comment:
So it can be used just like today's CompilerPipelineBuilder with `s/CompilerPipeline.Builder/PartiQLCompilerPipeline.legacy/`. For example,

```
//-
val pipeline = CompilerPipeline.Builder()....

//+
val pipeline = PartiQLCompilerPipeline.legacy()...
```

But to be honest it may not be needed. I could also create a legacy compiler which wraps Evaluating compiler. That way we could remove CompilerPipeline and CompilerPipeline.Builder yet keep the full functionality until the planner is ready

Reply:
Ok, I guess it was sort of weird imo that it returned a builder, while the `standard` function returned a `PartiQLCompilerPipeline` directly.